### PR TITLE
Create test_review.py

### PR DESCRIPTION
**Pull request by chamisfum**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Task reference

Please include task reference link and details (name, requirement, etc) here.

> **Example :** Create login page for student. [Task Reference](https://put_task_reference_link_here)

## Type of change

Please make any relevant options for the progress of your changes.

> Example for ceating login page task :
> - [x] Make login form
> - [ ] Make form validation for email
> - [ ] Make form validation for empty form

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] User Interface Local
- [ ] Postman Lesponse
- [ ] User Test

## Testing screenshot

Please provide us with your test results. This may be a positive or negative test for an API response. Also, provide us with the interface before and after your changes are applied if any. Provide a relevant title.

### Before my change

*Place your screenshot here*

### After my change

*Place your screenshot here*

# Checklist:

- [ ] My changes do not affect any errors in existing features
- [ ] Perform `git pull --rebase origin main` before ready to merge
- [ ] No conflict in my changes
- [ ] Any conflict are resolved
- [ ] I've follows [the code style guidelines](https://)
- [ ] I've follows [the code documentation guidelines](https://)
- [ ] I've added and run a test scenario that proves my changes are works
- [ ] I've provided  all relevant screenshots
- [ ] My changes generate no new warnings and no error
- [ ] Approve by at least  2 or more tech member
- [ ]  Approve by the project owner
- [ ] Using all relevant labels

### CODE REVIEW

**Review by DSC-UMM — COMMENTED** [review]
Leave a global comment to clarify any information to the PR owner. Also give a general feedback to the PR owner.

**Review by DSC-UMM — CHANGES_REQUESTED** [review]
Request any changes

File: test_review.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+print("hello dev!")
+print("please review my change
```
Comment (DSC-UMM) [inline test_review.py:2]: It seems you have typos in your code. Kindly check your second line

Comment (DSC-UMM) [inline test_review.py:2]: Kindly check your second line and make sure the syntax are same like the first one

Comment (DSC-UMM) [inline test_review.py:2]: It seems you have typos in your 2nd line. Kindly check the print syntax on line 2

**Review by DSC-UMM — APPROVED** [review]

File: test_review.py (hunk 1 above)
Comment (DSC-UMM) [reply test_review.py:2]: Well yeah you are right

**Review by DSC-UMM — COMMENTED** [review]
aa

**Review by ChamdaniMust — COMMENTED** [review]
Nothing much

**Pull request closed without merging**